lib/KeyboardAwareFlatList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import createReactClass from 'create-react-class';
+import PropTypes from 'prop-types';
+import { FlatList } from 'react-native';
+import KeyboardAwareMixin from './KeyboardAwareMixin';
+
+const KeyboardAwareFlatList = createReactClass({
+  propTypes: {
+    viewIsInsideTabBar: PropTypes.bool,
+    resetScrollToCoords: PropTypes.shape({
+      x: PropTypes.number,
+      y: PropTypes.number,
+    }),
+  },
+  mixins: [KeyboardAwareMixin],
+
+  componentWillMount: function () {
+    this.setViewIsInsideTabBar(this.props.viewIsInsideTabBar)
+    this.setResetScrollToCoords(this.props.resetScrollToCoords)
+  },
+
+  render: function () {
+    return (
+      <FlatList
+        ref='_rnkasv_keyboardView'
```

Comment:
@alvaromb How do you think about @chrisy-lili 's suggestion? We could have another PR for refs change.